footer.php

```diff
@@ -48,6 +55,28 @@
                 //end of all sidebar checks.
                 endif;?>
 
+                <div class="footer-social-icons">
+                    <a class="l-space" href="<?php echo get_option('facebook_url') ?>" style="color: <?php echo $primary_color ?>;">
+                    <i class="fa fa-facebook fa-1x" aria-hidden="true"></i>
+                    </a>
+                    <a class="l-space" href="<?php echo get_option('twitter_url') ?>" style="color: <?php echo $primary_color ?>;">
+                        <i class="fa fa-twitter fa-1x" aria-hidden="true"></i>
+                    </a>
+                    <a class="l-space" href="<?php echo get_option('instagram_url') ?>" style="color: <?php echo $primary_color ?>;">
+                        <i class="fa fa-instagram fa-1x" aria-hidden="true"></i>
+                    </a>
+                    <a class="l-space" href="<?php echo get_option('youtube_url') ?>" style="color: <?php echo $primary_color ?>;">
+                        <i class="fa fa-youtube-play fa-1x" aria-hidden="true"></i>
+                    </a>
+                    <a class="l-space" href="<?php echo get_option('linkedin_url') ?>" style="color: <?php echo $primary_color ?>;">
+                        <i class="fa fa-linkedin fa-1x" aria-hidden="true"></i>
+                    </a>
+                    <a class="l-space" href="<?php echo get_option('pinterest_url') ?>" style="color: <?php echo $primary_color ?>;">
+                        <i class="fa fa-pinterest-p fa-1x" aria-hidden="true"></i>
+                    </a>
+                </div>
```

Comment:
Make sure to reorder the icons by most popular to least based on a restaurant theme.